### DISCUSSION TITLE
Visual Code Output error

### DISCUSSION THREAD

**Discussion by prajjwallive** [microsoft/vscode-cpptools#7227] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76769901/112328290-4e58c180-8cde-11eb-9217-f34d727fab65.png)
Can see the output of the program in Visual C help!!

**Comment by sean-mcmanus** [comment, marked as answer]
It looks like you're just running gcc in the terminal and not running your program. Can you try the tutorial at https://code.visualstudio.com/docs/cpp/config-mingw ? And/or try using right-click "Build and Debug Active File"?
